### ISSUE TITLE
Battery status

### ISSUE THREAD

**Issue by Hiutale** [CongducPham/LowCostLoRaGw#242]
Hi. Is there any way to add battery level or voltage to transmit message? So you can check out when you have to change battery.

**Comment by CongducPham** [comment]
Yes, this is possible. There are many tutorial on the Internet to get the battery level from the Arduino board. You can do that and add a field to get this level on the cloud.

regards,
